service/src/main/java/bio/terra/axonserver/utils/CloudStorageUtils.java

```diff
@@ -25,6 +35,7 @@ public class CloudStorageUtils {
    * @param byteRange Byte range to read from the object
    * @return InputStream for the object content
    */
+  @Retryable(maxAttempts = 3, backoff = @Backoff(delay = 1000))
```

Comment:
Oh interesting, TIL. Ideally it would be nice to control which status codes are retryable (5xx) and which should fail immediately (404?), but this is okay for now.

Reply:
I'm with you, updated to exclude 404s for now.